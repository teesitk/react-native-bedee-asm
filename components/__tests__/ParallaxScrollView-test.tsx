import * as React from 'react';
import renderer from 'react-test-renderer';
import ParallaxScrollView from '../ParallaxScrollView';
import { ThemedView } from '../ThemedView';
import { ThemedText } from '../ThemedText';

jest.useFakeTimers()

it(`renders correctly`, () => {
  const tree = renderer.create(<ParallaxScrollView><ThemedView><ThemedText>xxx</ThemedText></ThemedView></ParallaxScrollView>).toJSON();

  expect(tree).toMatchSnapshot();
});
