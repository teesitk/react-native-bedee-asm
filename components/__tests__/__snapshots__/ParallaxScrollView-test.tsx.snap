// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      {
        "flex": 1,
      },
    ]
  }
>
  <RCTScrollView
    collapsable={false}
    forwardedRef={[Function]}
    jestAnimatedStyle={
      {
        "value": {},
      }
    }
    scrollEventThrottle={16}
  >
    <View>
      <View
        style={
          [
            {
              "backgroundColor": "#fff",
            },
            {
              "flex": 1,
              "gap": 16,
              "overflow": "hidden",
              "padding": 32,
            },
          ]
        }
      >
        <View
          style={
            [
              {
                "backgroundColor": "#fff",
              },
              undefined,
            ]
          }
        >
          <Text
            style={
              [
                {
                  "color": "#11181C",
                },
                {
                  "fontSize": 16,
                  "lineHeight": 24,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
              ]
            }
          >
            xxx
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
